javascript/javascript3/week3/screenshot_app.js

```diff
@@ -0,0 +1,121 @@
+const BASE_URL = "https://crudcrud.com/api/";
+const urlInput = document.getElementById("urlInput");
+const generateAndSaveScreenshotBtn =
+  document.getElementById("getScreenshotBtn");
+const savingMsg = document.getElementById("savingMsg");
+const displayListBtn = document.getElementById("showBtn");
+const savedImgsEl = document.getElementById("savedImgs");
+
+let screenshotURL;
```

Comment:
Since you are later checking if something is inside screenshotURL I would advice using null as a starting value. It's just easier to check imo then a undefined or a empty string